### DISCUSSION TITLE
Run only MemoryDiagnoser, not performance diagnoser (warmup etc.) - to speed up execution

### DISCUSSION THREAD

**Discussion by MihaMarkic** [dotnet/BenchmarkDotNet#2451] [Q&A]
Hi guys,

What would be the best way to avoid warmups and whatnot when only memory diagnostics is required?

**Comment by timcassell** [comment]
I would _like_ to say dry job, but due to tiered jit, it can give inaccurate results (see #1962). I recommend short job to get accurate results quickly. If your benchmark is long-running, such that it executes fewer than ~30 times, even short job won't give you accurate results (unless you actually want to measure tier0 allocations). You could also try disabling tiered jit, which I haven't tried myself.

**Reply by MihaMarkic** [reply]
Yep, that's what I was thinking so far. My benchmarks are really quick, so I'll try with short jobs.
